app/modules/intelligence/prompts_provider/base_prompts_provider.py

```
@@ -0,0 +1,72 @@
+
+      
+class BasePromptsProvider(ABC):
+    PROMPTS: Dict = {}
+    CLASSIFICATION_PROMPTS: Dict = {}
+    AGENTS_DICT: Dict[str, Dict[str, str]] ={}
+    TASK_PROMPTS: Dict[str, Dict[str, str]] = {}
+
+    @classmethod
+    async def get_prompts(
+        cls, agent_id: str,
+        prompt_types: List[PromptType] = [PromptType.SYSTEM],
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)